be/src/pipeline/exec/exchange_sink_buffer.cpp

```diff
@@ -247,6 +247,9 @@ Status ExchangeSinkBuffer::_send_rpc(InstanceLoId id) {
         _instance_to_rpc_ctx[id] = rpc_ctx;
 
         closure->cntl.set_timeout_ms(request.channel->_brpc_timeout_ms);
+        if (config::tablet_writer_ignore_eovercrowded) {
```

Comment:
It is this problem, but please add another parameter because tablet_writer_ignore_eovercrowded is used for tablet sink.